build/bootstrap_mcs.sh

```diff
@@ -529,6 +533,8 @@ generate_svgs()
     fi
 }
 
+disable_git_restore_frozen_revision
```

Comment:
This might be unwanted behavior to be honest. @mariadb-LeonidFedorov  what do you think?